src/WebJobs.Extensions.CosmosDB/WebJobs.Extensions.CosmosDB.csproj

```diff
@@ -10,11 +10,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.DocumentDB.ChangeFeedProcessor" Version="1.3.3" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.0-beta7-11385" />
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
   </ItemGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\WebJobs.Extensions\WebJobs.Extensions.csproj" />
```

Comment:
I removed the dependency on the Extensions project from all of these. It used to exist because many of the extensions were using shared binding provider base classes, etc. but that is no longer the case.